pkg/manifest/manifest.go

```diff
@@ -16,24 +16,27 @@ type Data struct {
 }
 
 // FromStringData Returns manifests from string data
-func FromStringData(data string) []Data {
+func FromStringData(data string) ([]Data, error) {
 	manifests := make([]Data, 0)
 	err := yaml.Unmarshal([]byte(data), &manifests)
 	if err != nil {
-		return manifests
+		return manifests, err
 	}
 
-	return manifests
+	return manifests, nil
 }
```

Comment:
I think here we can just `return manifests, err` - same result. We cannot return `nil` when `err != nil` as it is used later

Reply:
Agree. See my answer on your other comment.